src/gui/mzroll/pollyintegration.cpp

```diff
@@ -0,0 +1,263 @@
+#include "pollyintegration.h"
+
+PollyIntegration::PollyIntegration(TableDockWidget* tableDockWidget)
+{
+
+    _tableDockWidget = tableDockWidget;
+}
+
+PollyIntegration::~PollyIntegration()
+{
+    qDebug()<<"exiting PollyIntegration now....";
+}
+
+QByteArray PollyIntegration::run_qt_process(QString command){
+    QProcess process;
+    process.start(command);
+    process.waitForFinished();
+    QByteArray result = process.readAllStandardOutput();
+    QByteArray result2 = process.readAllStandardError();
```

Comment:
What's the use of **result2**

Reply:
In case of process Error we can have a look at this variable..
This is not being used in the function as of Now, but we should keep it there for future references as we want to terminate the uploading process in case of Process error..